Source/scrollrt.cpp

```diff
@@ -207,9 +207,7 @@ void DrawMissile(int x, int y, int sx, int sy, BOOL pre)
 	}
 
 	for (i = 0; i < nummissiles; i++) {
-		/// ASSERT: assert(missileactive[i] < MAXMISSILES);
-		if (missileactive[i] >= MAXMISSILES)
-			break;

```

Comment:
Im curious if this is what you meant by improving performance in releases. Because this removes the safety check altogether. The debug release used an assertion, but release builds just break out.

Reply:
Yeah not having the checks will make it a tiny bit faster, since we will die because of the assertions I don't think we need to also need the safety checks